user_guide/data_management_tags.rst

```diff
@@ -0,0 +1,152 @@
+.. _user-guide-tags:
+
+Tags
+====
+
+Tag is non-hierarchical keyword or phrase assigned to a record. First of all, tags provide additional information about 
+the records to all the system users. Tags can be assigned to any records of any entity and used for  
+:ref:`filtering <user-guide-filters-management>`, they can be used to create a report only for the records that 
+have/don't have a specific tag and generate targeted segments for mailings and marketing campaigns.
+Moreover, you can easily find all the records with a specific tag, even not knowing 
+their other details from anywhere in the system as described hereunder.  
```

Comment:
Edit: Tags are non-hierarchical keywords or phrases that are assigned to records. They provide additional information about records to all the system users. Tags can be assigned to any records of any entity and used for :ref:`filtering <user-guide-filters-management>`. They can also be used to create a report for only the records that have/don't have a specific tag, and even generate targeted segments for mailings and marketing campaigns. Moreover, you can easily find all records that have a specific tag from anywhere in the system, even if you don't know their other details, as described below.